mountpoint-s3/src/main.rs

```diff
@@ -259,6 +261,22 @@ struct CliArgs {
         help_heading = ADVANCED_OPTIONS_HEADER,
     )]
     pub user_agent_prefix: Option<String>,
+
+    #[cfg(feature = "sse_kms")]
```

Comment:
should we have 2 different options for dsse-kms and sse-kms (mutually exclusive)?

Reply:
I believe you are suggesting here to alter the CLI interface, so that we have 2 flags `dsse-kms` and `sse-kms` instead of a single one `server_side_encryption`; I personally think that having a single flag is more intuitive and it complies with `aws s3api put-object` cli interface, but I'll bring up this discussion on the next meeting